testbed/stack.ts

```diff
@@ -12,7 +12,7 @@ export interface TestbedProps extends cdk.StackProps {
 
 export class Testbed extends cdk.Stack {
     constructor(scope: cdk.Construct, id: string="testbed", props: TestbedProps) {
-        super(scope, id, props)
+        super(scope, id)
```

Comment:
With this change will we be able to deploy multiple testbeds in the same region in one account? IIUC, this includes the CFN stack name but maybe I am missing something here.